e2e/commands_test.go

```diff
@@ -123,28 +123,28 @@ maintainers:
 	assert.Assert(t, fs.Equal(tmpDir.Join(dirName), manifest))
 
 	// validate metadata with JSON Schema
-	cmd.Command = []string{dockerApp, "validate", testAppName}
+	cmd.Command = dockerCli.Command("app", "validate", testAppName)
 	icmd.RunCmd(cmd).Assert(t, icmd.Success)
 
 	// test single-file init
-	cmd.Command = []string{dockerApp,
+	cmd.Command = dockerCli.Command("app",
 		"init", "tac",
-		"-c", tmpDir.Join(internal.ComposeFileName),
-		"-d", "my cool app",
-		"-m", "bob",
-		"-m", "joe:joe@joe.com",
-		"-s",
-	}
+		"--compose-file", tmpDir.Join(internal.ComposeFileName),
+		"--description", "my cool app",
+		"--maintainer", "bob",
+		"--maintainer", "joe:joe@joe.com",
+		"--single-file",
```

Comment:
These are due to https://github.com/docker/cli/issues/1699 I think?

You didn't actually remove the short versions so there is no followup to do when that is fixed, right?

Reply:
Yep, that's correct